new-lamassu-admin/src/pages/Blacklist/Blacklist.styles.js

```diff
@@ -0,0 +1,42 @@
+export default {
+  grid: {
+    flex: 1,
+    height: '100%'
+  },
+  content: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1,
+    marginLeft: 48
+  },
+  bottomRight: {
+    marginLeft: 'auto',
+    marginTop: 'auto',
+    marginBottom: '24px'
+  },
+  modalTitle: {
+    lineHeight: '120%',
+    color: '#1b2559',
+    fontSize: 14,
+    fontFamily: 'Mont',
+    fontWeight: 900
+  },
+  subtitle: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    flexDirection: 'row'
+  },
+  white: {
+    color: 'white'
+  },
+  deleteButton: {
+    paddingLeft: '13px'
```

Comment:
You can shorten this to

`paddingLeft: 13` you can use numbers instead of strings with `px` at the end.